src/main/java/com/marklogic/appdeployer/command/hosts/AssignHostsToGroupsCommand.java

```diff
@@ -0,0 +1,93 @@
+package com.marklogic.appdeployer.command.hosts;
+
+import java.util.Map;
+
+import com.marklogic.appdeployer.command.AbstractUndoableCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.mgmt.admin.ActionRequiringRestart;
+import com.marklogic.mgmt.resource.appservers.ServerManager;
+import com.marklogic.mgmt.resource.hosts.HostManager;
+
+public class AssignHostsToGroupsCommand extends AbstractUndoableCommand {
+
+	private static final String DEFAULT_GROUP_NAME = "Default";
+	private static final String MANAGE_SERVER_NAME = "Manage";
+	private static final String MANAGE_SERVER_JSON_TEMPLATE = "{\"server-name\":\"%s\", \"group-name\":\"%s\", \"server-type\":\"http\", \"root\":\"Apps/\", \"port\":8002, \"content-database\":\"App-Services\", \"error-handler\":\"manage/error-handler.xqy\", \"url-rewriter\":\"manage/rewriter.xqy\", \"privilege\":\"http://marklogic.com/xdmp/privileges/manage\"}";
```

Comment:
Let's do the same approach here for the Manage and App-Services server objects - create a private variable for each, and have the constructor of the command configure each Server object. Then add a setter for each one so that a client again has control over how these objects are configured.